config/routes.rb

```diff
@@ -13,5 +13,18 @@
       resources :likes, only: [:create, :destroy]
     end
   end
```

Comment:
[OPTIONAL]
- [ ] Also make sure you don't have the `views` endpoints as they it's not necessary to have them in a `rails backend application`